analytics_data_api/constants/learner.py

```diff
@@ -0,0 +1 @@
+LEARNER_API_ROSTER_PAGE_SIZE = 25
```

Comment:
What do you think about updating this to `DEFAULT_LEARNER_API_ROSTER_PAGE_SIZE`?